src/Recommendations.js

```diff
@@ -51,12 +53,22 @@ const buildSearchParamsFromRecommendations_TEMPORARY_BETA = (record, props) => {
       hitsPerPage = record.recommendations.length;
     }
   }
-
+  if (this.model === 'bought-together') {
+    return {
+      optionalFilters: [],
+      filters:
+        (recoFilters.length > 0 &&
+          composeFilters(recoFilters)) ||
+        null,
```

Comment:
```suggestion
      filters: composeFilters(recoFilters) || null,
```

The `recoFilters.length > 0` test should already be covered by your logic above; isn't it?